modules/Rubrics/rubrics_addProcess.php

```diff
@@ -54,15 +50,13 @@
             } else {
                 $gibbonDepartmentID = null;
             }
-            $name = $_POST['name'];
-            $active = $_POST['active'];
-            $category = $_POST['category'];
-            $description = $_POST['description'];
-            $gibbonYearGroupIDList = isset($_POST['gibbonYearGroupIDList']) ? implode(',', $_POST['gibbonYearGroupIDList']) : '';
-            $gibbonScaleID = null;
-            if ($_POST['gibbonScaleID'] != '') {
-                $gibbonScaleID = $_POST['gibbonScaleID'];
-            }
+            $name = $_POST['name'] ?? '';
+            $active = $_POST['active'] ?? '';
+            $category = $_POST['category'] ?? '';
+            $description = $_POST['description'] ?? '';
+            $gibbonYearGroupIDList = implode(',', $_POST['gibbonYearGroupIDList']) ?? '';
```

Comment:
👀  Keep an eye on where you're applying the null coalesce. Also, by moving the null coalesce into the implode function, which expects and array, the null coalesce value should be an array.

```suggestion
            $gibbonYearGroupIDList = implode(',', $_POST['gibbonYearGroupIDList'] ?? []);
```